shell/platform/windows/BUILD.gn

```diff
@@ -21,6 +21,11 @@ config("relative_flutter_windows_headers") {
   include_dirs = [ "public" ]
 }
 
+# When OS target is WinUWP use FLUTTER_WINUWP as the preprocessor flag to conditionalize
+config("flutter_windows_winuwp_defs") {
+  defines = [ "FLUTTER_WINUWP 1" ]
```

Comment:
I've updated the name and value but moving it to build/config/win/BUILD.gn means that it's a dependent change in buildroot that would have to land ahead of this change.  I would rather land without moving since there will have to be a subsequent change to buildroot with other changes and i'd rather not complicate the dependencies further.  I've added a TODO to that effect, Please reopen if you disagree.